cli/file_fetcher.rs

```diff
@@ -1262,9 +1262,13 @@ mod tests {
       let r = fetcher.fetch_source_file(&specifier);
       assert!(r.is_err());
 
-      // Assuming cwd is the deno repo root.
+      let p = std::path::PathBuf::from(env!("CARGO_MANIFEST_DIR"))
+        .parent()
+        .unwrap()
+        .join("js/main.ts")
+        .to_owned();
       let specifier =
-        ModuleSpecifier::resolve_url_or_path("js/main.ts").unwrap();
+        ModuleSpecifier::resolve_url_or_path(&p.to_string_lossy()).unwrap();
```

Comment:
Generally prefer `p.to_str().unwrap()` as that will panic instead of discarding characters if exact conversion is not possible.